### ISSUE TITLE
Package maintaining

### ISSUE THREAD

**Issue by haoadoreorange** [ivogabe/gulp-typescript#664]
Hi,
I notice that this package seems not maintained anymore. Are @ivogabe still there ? I can maintain it if needed. Asking this before forked.

**Comment by ivogabe** [comment]
Hi, I'm still here! It became very difficult for me to maintain this package; I was always hoping that I would get more time to work on this soon but I always ended up being busy with other things. I barely use TypeScript for work nowadays (working in Haskell now) so I don't really have time nor work-related reasons to work on this.

It would be great if you could maintain it! We can discuss things like the structure or ideas behind the project if you want.

**Comment by ivogabe** [comment]
Though I must warn you that this plugin is actually quite complex, as there are many different configurations in which users want to use it. I think the most annoying thing is the fact that TypeScript and gulp both have their own ideas of handling paths which may conflict each other (gulp files have a base directory, and TypeScript has various path-related options)

**Comment by haoadoreorange** [comment]
Thank you for responding quickly ! 
Actually I noticed a bug/behavior that is inconsistent with tsc so I dig around to fix it and that helped graping a first idea of what the plugin look like. And sure It would be great if we can discuss about the structure ! it'd be easier for me. How do we proceed ?

**Comment by ivogabe** [comment]
Great, we could discuss that in a chat or a call. What would you prefer?

**Comment by ivogabe** [comment]
Or we could just discuss it in this issue of course, then it's also documented for other people who are interested.

**Comment by haoadoreorange** [comment]
Tracking down #666 now I'm fairly confident being able to take over maintenance. How do we proceed ?

**Comment by ivogabe** [comment]
Nice! You could send PRs, or I can transfer the repository to you if you feel confident enough?

**Comment by haoadoreorange** [comment]
Maybe add me as collaborator is enough ? You should still maintain the ownership imho xD

**Comment by ivogabe** [comment]
You should have an invite now! If you're not absolutely sure about some changes, you can also send them as a PR and I'll take a look.

**Comment by ivogabe** [comment]
Some first steps might be to look at the JSON issue and update the dependencies. Long term could be looking at incremental compilation and improving that.

**Comment by StanislavKharchenko** [comment]
So, what with project status?
@ivogabe @haoadoreorange Do you abandon it and we need think about replace of this utility?

**Comment by haoadoreorange** [comment]
@StanislavKharchenko Hello, I personally still use it for some of my project, it just hasn't been updated for a while. Are you currently using a fork with your patches ?

**Comment by StanislavKharchenko** [comment]
@haoadoreorange No, we also using origin npm package. The question was since the latest version is not compatible with NodeJS 18.
To avoid this, we added a peer dependency to update "source-map": "^0.8.0-beta.0"

**Comment by albertodiazdorado** [comment]
Do you know if there is an alternative to this package that undergoes active development and maintenance?
